packages/oas/utils/json-schema-to-typescript-parsers/yamlParser.ts

```diff
@@ -1,8 +1,8 @@
-import { FileInfo } from 'json-schema-ref-parser';
-import JsonSchemaRefParser from 'json-schema-ref-parser';
+import { FileInfo } from '@apidevtools/json-schema-ref-parser';
+import JsonSchemaRefParser from '@apidevtools/json-schema-ref-parser';
+import { SchemaObject } from 'ajv';
 
-import { JSONSchema4 } from './JSONSchema4';
-import { applyDateFormat } from './applyDateFormat';
+import { applyDateFormat } from '../applyDateFormat';
 
 // NOTE: this is the yaml parser copied from v6.1.0 of json-schema-ref-parser (and modified) since it is not exported
```

Comment:
```suggestion
// NOTE: this is the yaml parser copied from v9.0.9 of json-schema-ref-parser (and modified) since it is not exported
```
maybe also include this link somewhere in the comment? https://github.com/APIDevTools/json-schema-ref-parser/blob/v9.0.9/lib/parsers/yaml.js